ansible/hosts.ini

```diff
@@ -0,0 +1,13 @@
+# Groups of hosts
+
+# Roles
+
+[jenkins-master]
+host-os-jenkins.aus.stglabs.ibm.com
```

Comment:
This is an internal host.

What if we listed a sample host like `host-os-jenkins.domain.tld`?